pyradiosky/skymodel.py

```diff
@@ -3512,6 +3702,11 @@ def write_skyh5(
                 val = param.value
                 parname = param.name
 
+                if parname == "lon":
+                    parname = "ra"
+                if parname == "lat":
+                    parname = "dec"
+
```

Comment:
we should just set the parname as whatever the name is. this is a holdover from when we only supported icrs

Reply:
Do we have a document describing `skyh5` yet? I didn't want to make changes to the skyh5 format without making sure it's documented accordingly.

Reply:
searching through the docs. I don't see one. But possible I'm just blind.

Reply:
#118 is still open